passes/techmap/abc9.cc

```diff
@@ -151,10 +151,14 @@ struct Abc9Pass : public ScriptPass
 		log("        generate netlist using luts. Use the specified costs for luts with 1,\n");
 		log("        2, 3, .. inputs.\n");
 		log("\n");
+		log("    -lutlib\n");
+		log("        generate netlist using luts. Use the modules that are loaded into the\n");
+		log("        current design and have the (* abc9_lut *) attribute for the library of\n");
+		log("        the available lut primitives.\n");
+		log("\n");
 		log("    -maxlut <width>\n");
-		log("        when auto-generating the lut library, discard all luts equal to or\n");
-		log("        greater than this size (applicable when neither -lut nor -luts is\n");
-		log("        specified).\n");
+		log("        when sourcing the lut library from the current design (option -lutlib),\n");
+		log("        discard all lut primitives with greater than the specified width.\n");
```

Comment:
Btw the old help text said `-maxlut` means "discard greater than or equal", but as far as I can tell "discard greater than" is what's right